routes/sapiV1CapiV2.js

```diff
@@ -129,4 +130,129 @@ router.get('/test', async (req, res, next) => {
 	});
 });
 
+function prepAnnotationsGroup( groupName, annoPairs, groupDetails, searchResponse ){
+  const group = {
+    name : groupName,
+    byCount : {
+      topAnnotations : [],
+      annotationsBubblingUnder : [],
+    },
+  };
+
+  group.byCount.topAnnotations = annoPairs
+  .filter( pair => { return pair[1] > 1; }) // just those with count > 1
+  .map( pair => { // bring together details, incl list of articles
+    const name      = pair[0];
+    const count     = pair[1];
+    const uuids     = groupDetails.uuidsGroupedByItem[name];
+    const articles  = uuids.map( uuid => { return searchResponse.articlesByUuid[uuid]; });
+    articles.forEach( article => {
+      if (article.mainImage
+        && article.mainImage.members
+        && article.mainImage.members.length > 0 ) {
+          article.mainImage.thumbnailUrl = image.formatImageUrl(article.mainImage.members[0], 200);
+      }
+    });
+
+    return {
+      name,
+      count,
+      uuids,
+      articles,
+    }
+  })
+  ;
+
+  group.byCount.annotationsBubblingUnder = annoPairs
+  .filter( pair => { return pair[1] == 1; })
```

Comment:
Similarly, `parseInt()` and use a strict equality

Reply:
=== ok, but should always be an int